config/roguelike_dungeons/settings/loot_dungeon_desert.json

```diff
@@ -0,0 +1,88 @@
+{
+"name": "loot_desert",
+"loot_rules": [
+	{"level": [5], "type": "FOOD", "each": true, "quantity": 1, "loot": [{"data": {"name": "minecraft:dye", "meta": 4, "min": 2, "max": 3}}]},
+	{"level": [5], "type": "FOOD", "each": true, "quantity": 1, "loot": [{"data": {"name": "minecraft:dye", "meta": 4, "min": 2, "max": 3}}]},
+	{"level": [5], "type": "FOOD", "each": true, "quantity": 4, "loot": [
```

Comment:
The loot_dungeon_*.json files all start with these loot rules giving dyes and applying to level 5. I thought level 5 didn't exist. Are those valid and necessary?